test/lib/register_loader.js

```diff
@@ -37,19 +37,47 @@ describe('registerLoader', function() {
       registerLoader(app, extensions, configPath);
     });
 
+    it('Should emit only a "require" event when registering loader ' +
```

Comment:
if you add an `.eslintrc` in the test directory that extends `gulp/test`, you don't need to worry about long test names.